src/test/components/__snapshots__/Home.test.js.snap

```diff
@@ -116,20 +118,16 @@ exports[`app/Home renders the information screen for other browsers 1`] = `
             <span
               class="homeSectionDocsIcon"
             />
-            Documentation
+            <span>
+              Documentation
+            </span>
           </a>
           <h2>
             How to view and record profiles
           </h2>
           <p>
-            Recording performance profiles requires
-             
-            <a
-              href="https://www.mozilla.org/en-US/firefox/new/"
-            >
-              Firefox
-            </a>
-            . However, existing profiles can be viewed in any modern browser.
+            Recording performance profiles requires &lt;a&gt;Firefox&lt;/a&gt;. 
+                However, existing profiles can be viewed in any modern browser.
```

Comment:
Looks like the link is missing here.

Reply:
I don't understand how the snapshots are working with `Localized`. It's clear that the texts are not being replaced with FTL strings. I wonder why :/